caching-attribute-service-client/gradle.lockfile

```diff
@@ -0,0 +1,42 @@
+# This is a Gradle generated file for dependency locking.
```

Comment:
Should this file be checked in?

Reply:
So based on GHAs it's the whole point of the PR.

Reply:
yep (also see line 3 😉 )